package.json

```diff
@@ -17,6 +17,7 @@
   "author": "Guyon Roche",
   "license": "MIT",
   "dependencies": {
+    "async": "^2.6.0",
```

Comment:
Please don't add to "dependencies" for things that will only be used by the system test